frontend/src/metrics.js

```diff
@@ -1,4 +1,4 @@
-const testPilotGA = require('testpilot-ga');
+const testPilotGA = require('testpilot-ga/src/TestPilotGA');
```

Comment:
It technically _works_, but only due to a bug in https://github.com/mozilla/testpilot-ga/pull/13.
Not sure how you want to address. We'd either have to re-compile the **testpilot-ga** dist files, or kill my upstream PR and remove the bad `files` in **testpilot-ga** repo.